_data/events/bonn.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Global Day Of Code Retreat at itemis Bonn",
+  "url": "https://www.meetup.com/de-DE/preview/itemis/events/244448511",
```

Comment:
This looks like you accidentally used the preview link for the event on Meetup. Do you want to swap it out for the correct one?

Reply:
I think it's the right url. I modified the url in the browser and removed "/preview" but meetup forwarded me to the same url (with "/preview") again. Thanks for asking!